src/typecheck.c

```diff
@@ -87,9 +87,8 @@ static const char *short_type_description(const Type *t)
         case TYPE_ARRAY:
             return "an array type";
         case TYPE_BOOL:
-            // TODO: Is it possible to get this in an error message?
-            return "the built-in bool type";
-    }
+            assert(0);
```

Comment:
It is actually possible to get this in an error message:
```
def foo() -> None:
    x = True.asdf()
```

Reply:
Fine, I never thought user may run the code like this.